### PR TITLE
Update kubectl-overview.md

### PR DESCRIPTION
**Pull request by techtonik**
Add link to kubectl installation instructions

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1417)

### CODE REVIEW

**Comment by bgrant0607** [comment]
@devin-donnelly Is the correct form for relative links documented somewhere?

**Comment by techtonik** [comment]
@bgrant0607 are there any CI tests for documentation to catch those conventions and point to docs automatically?

**Comment by bgrant0607** [comment]
@techtonik Link checking is an open issue: #310

The checker from the main repo needs to be modified in order for it to work with the different conventions here, if you want to help:
https://github.com/kubernetes/kubernetes/blob/master/cmd/mungedocs/links.go

**Comment by techtonik** [comment]
It looks like the tool invoked is https://github.com/kubernetes/md-check however, I don't see any docs for it or explanations how it integrates with `mungedocs` and the `links.go` file above.

**Comment by techtonik** [comment]
I also thought for some reason that this `diff check` is a separate service deployed on kubernetes. not a Travis job. =)

File: docs/user-guide/kubectl-overview.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ assignees:
 
 ---
 
-Use this overview of the `kubectl` command line interface to help you start running commands against Kubernetes clusters. This overview quickly covers `kubectl` syntax, describes the command operations, and provides common examples. For details about each command, including all the supported flags and subcommands, see the [kubectl](/docs/user-guide/kubectl) reference documentation.
+`kubectl` is a command line interface for running commands against Kubernetes clusters. This overview quickly covers `kubectl` syntax, describes the command operations, and provides common examples. For details about each command, including all the supported flags and subcommands, see the [kubectl](/docs/user-guide/kubectl) reference documentation. For installation instructions see [prerequisites](/docs/user-guide/prereqs).
```
Comment (devin-donnelly) [inline docs/user-guide/kubectl-overview.md]: "quickly" is unncessary in the second sentence.

**Comment by devin-donnelly** [comment]
One nit. Docs LGTM otherwise.

**Comment by techtonik** [comment]
@devin-donnelly done.

**Pull request merged**